hypervisor/network/network_linux.go

```diff
@@ -1049,45 +1049,27 @@ func AllocateAddr(requestedIP string) (*Settings, error) {
 		return nil, err
 	}
 
-	maskSize, _ := BridgeIPv4Net.Mask.Size()
-
 	mac, err := GenRandomMac()
 	if err != nil {
 		glog.Errorf("Generate Random Mac address failed")
 		return nil, err
 	}
 
 	return &Settings{
-		Mac:         mac,
-		IPAddress:   ip.String(),
-		Gateway:     BridgeIPv4Net.IP.String(),
-		Bridge:      BridgeIface,
-		IPPrefixLen: maskSize,
-		Device:      "",
-		File:        nil,
-		Automatic:   true,
+		Mac:       mac,
+		IPAddress: []string{ip.String()},
+		Mtu:       DefaultMtu,
```

Comment:
I think by default we should not care about mtu, vm/os will setup the appropriate mtu for us. lets just set it null.